near-workspaces/utils/dummyData.ts

```diff
@@ -0,0 +1,80 @@
+// Amounts
+export const ONE_NEAR = '1000000000000000000000000'
+export const ONE_YOCTO = '1'
+export const CONTRACT_MINT_PRICE = ONE_NEAR
+export const CONTRACT_MINT_GAS = '300000000000000'
+
+
+// Utility
+export function randomString(length) {
+    let result = ''
+    const characters =
+        'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+    const charactersLength = characters.length
+    for (let i = 0; i < length; i++) {
+        result += characters.charAt(
+            Math.floor(Math.random() * charactersLength)
+        )
+    }
+    return result
+}
+
+export function randomInt(min = 0, max = 10) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+
+// Data
+export const CONTRACT_METADATA = {
+    spec: 'nft-1.0.0',
+    name: 'nftExample',
+    symbol: 'NFTEXAMPLE',
+    icon: '',
+    base_uri: 'https://picsum.photos',
+    reference: '',
+    reference_hash: '',
+    packages_script: '',
+    render_script: '',
+    style_css: '',
+    parameters: '',
+}
+
+export const CONTRACT_EXTRA = {
+    mint_price: CONTRACT_MINT_PRICE,
+    max_copies: 100,
+    default_max_len_payout: 20,
+    mints_per_address: 50,
+    mint_payee_id: 'jenny.test.near',
+    mint_royalty_id: 'jenny.test.near',
+    mint_royalty_amount: 10,
+}
+
+export function TOKEN_METADATA() {
```

Comment:
I understand it's a const and that's why the upper case but I would just keep it: `get_random_token_metadata`. Same for `BID` and the others, if there is any somewhere else.